scripts/obtain/uninstall/dotnet-uninstall-pkgs.sh

```diff
@@ -17,6 +17,9 @@ dotnet_pkg_name_suffix="com.microsoft.dotnet"
 dotnet_install_root="/usr/local/share/dotnet"
 dotnet_path_file="/etc/paths.d/dotnet"
 
+# nuget paths
+nuget_root=~/.nuget
```

Comment:
Yeah. Though, it should all expand with "~". But, better safe than 3x sorry. :smile: